R/pkg/tests/fulltests/test_utils.R

```diff
@@ -89,7 +89,10 @@ test_that("cleanClosure on R functions", {
     lapply(x, g) + 1  # Test for capturing function call "g"'s closure as a argument of lapply.
     l$field[1, 1] <- 3  # Test for access operators `$`.
     res <- defUse + l$field[1, ]  # Test for def-use chain of "defUse", and "" symbol.
-    f(res)  # Test for recursive calls.
+    # Enable once SPARK-30629 is fixed
```

Comment:
That's clear bug, trivial to reproduce ([SPARK-30629](https://issues.apache.org/jira/browse/SPARK-30629)). Not sure why I didn't fail on `testthat` 1.x.